boards/firefly/Makefile

```diff
@@ -0,0 +1,5 @@
+MODULE = board
+
+DIRS = $(RIOTBOARD)/remote-common
```

Comment:
Event if it's not the case in other places, I'm wondering if using `DIRS +=` wouldn't be better ?

Reply:
I agree, shouldn't do any harm. I will change it.